app/models/sensor_type.rb

```diff
@@ -1,4 +1,5 @@
 class SensorType < ActiveRecord::Base
   default_scope { order(:property) }
   validates :unit, length: {minimum:  0, allow_nil: false}
+  enum data_collection_method: {automatic: 0, manual: 1}
```

Comment:
alright :laughing: `data_collection_method` is the actual attribute.. still you can do the ordering with sql